### ISSUE TITLE
home subfolders icons

### ISSUE THREAD

**Issue by lo-yak** [thesquash/gtk-theme-raleigh#10]
hi there! thanks for this amazing theme.
i'm using it on xfce, the theme however doesn't seems to have a specific icon for the home subfolders (music, videos, pictures, ecc.) and for them it relies on the fallback theme which in my case is papirus but, since your theme is very unique, it's very ugly. i find it strange since the desktop icon is specific to desktop (it has a pencil and a paper on it), but it's the only one.
so don't icons for the subfolders exist? if not, how can i set the "normal" folder icon for subfolders too so at least they will be using all the same theme?

**Comment by thesquash** [comment]
Thanks for reporting this issue!

The original GNOME icon theme (from which I extracted most of the icons in this icon theme) did not have special icons for, say, the Documents, Music, and other sub-folders of /home -- at least, I don't think the GNOME icon theme did, not this very old version anyway.  Considering that apparently GNOME didn't have the icons back then, I didn't think that I needed to add icons for such subfolders to my icon theme.  I would have added links to the generic "folder" icon, but on my primary desktop environment (MATE) apparently that is not necessary, so I never bothered.

But yeah, there's a lot of broken and missing icons in this theme on XFCE.  I've been alternating between using XFCE and using MATE as my desktop environment though, which means that I too am getting annoyed by all the missing icons, so I'm working on adding more icons and links to icons.  (That is why there are a few new icons specific to XFCE.  I just haven't done them all yet.)

So in other words, thanks for reporting this.  Even if I was going to do something myself, having somebody else come and tell me about it is good.  Normally I just get around to doing stuff when I can, but comments like these remind me that there are other people using this theme.  And that is a good feeling.

**Comment by lo-yak** [comment]
Hi man, thanks for the reply.

> comments like these remind me that there are other people using this theme.

Well, I don't know about people, but I can speak for me: I searched a lot for a good dark theme for Xfce and I get the peace of mind when I found your theme. I can say this is my favourite GTK theme hands down!
So yes, I'm using it and I **really** care about it. If it wasn't for it I'd be still wandering and frustrating changing theme every day :)
The only problem with it is that, being pretty original, I can't find an icon theme that sticks well with it for my taste, except yours, and I have a bit of OCD for these things, so I suffer a bit every time I open my home folder lol

So it is better used on MATE? Didn't know that, I never used it extensively MATE and I didn't saw too much stuff in the MATE folder.
As for the other icons missing, I inherit Papirus-Dark (and commented out your volume icons tbh, which are very small in my tray for some reason) and, besides the home subfolders, it is a pretty decent replacement, even though would be nice to have more icons!
What icon theme do you inherit for the missing icons? Do you have some suggestions?

Anyway for a temporary workaround I could link to the generic folder icon manually, like you meant to do. It is just a symlink? To what icon, precisely? I tried to do it with the /generic//48/folder.png icon, but it doesn't seemed right to me so I thought maybe I picked the wrong one. If you could tell me exactly how to do it would be great.

That said, if I can help you anyhow to carry on the work, I'd be very happy to do it! So if there's something I could do just ask me :)
